Reminder/settings.py

```diff
@@ -37,9 +37,19 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
-    'blog',
+    'rest_framework',
+    'blog.apps.BlogConfig',
+    'rest_blog.apps.RestBlogConfig',
```

Comment:
@vamsikrishna-kolli This is good. You can do this also. But in this situation, you don't have any particular Application configurations. So you can remove the `apps.py` file. And then just put it as 'blog' and  'rest_blog'.